bundles/org.openhab.voice.watsonstt/README.md

```diff
@@ -0,0 +1,63 @@
+# IBM Watson Speech-to-Text
+
+Watson STT Service uses the non-free IBM Watson Speech-to-Text API to transcript audio data to text. 
+Be aware, that using this service may incur cost on your IBM account.
+You can find pricing information on [its page](https://www.ibm.com/cloud/watson-speech-to-text/pricing).
+
+## Obtaining Credentials
+
+Before you can use this add-on, you should create a Speech-to-Text instance in the IBM Cloud service. 
+
+* Go to the following [link](https://cloud.ibm.com/catalog/services/speech-to-text) and create the instance in your desired region.
```

Comment:
Are we talking about instances that will be deleted in 2022 ?

Reply:
No, I'm using one and I can't see any notice about the service going down.